WordPress/src/main/java/org/wordpress/android/datasets/PeopleTable.java

```diff
@@ -0,0 +1,119 @@
+package org.wordpress.android.datasets;
+
+import android.content.ContentValues;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+
+import org.wordpress.android.WordPress;
+import org.wordpress.android.models.Person;
+import org.wordpress.android.models.Role;
+import org.wordpress.android.util.AppLog;
+import org.wordpress.android.util.SqlUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PeopleTable {
+    public static final String PEOPLE_TABLE = "people";
+
+    private static SQLiteDatabase getReadableDb() {
+        return WordPress.wpDB.getDatabase();
+    }
+    private static SQLiteDatabase getWritableDb() {
+        return WordPress.wpDB.getDatabase();
+    }
+
+    public static void createTables(SQLiteDatabase db) {
+        db.execSQL("CREATE TABLE " + PEOPLE_TABLE + " ("
+                + "person_id               INTEGER DEFAULT 0,"
+                + "local_blog_id           INTEGER DEFAULT 0,"
+                + "user_name               TEXT,"
+                + "first_name              TEXT,"
+                + "last_name               TEXT,"
+                + "display_name            TEXT,"
+                + "avatar_url              TEXT,"
+                + "role                    TEXT,"
+                + "PRIMARY KEY (person_id, local_blog_id)"
+                + ");");
+    }
+
+    private static void dropTables(SQLiteDatabase db) {
+        db.execSQL("DROP TABLE IF EXISTS " + PEOPLE_TABLE);
+    }
+
+    public static void reset(SQLiteDatabase db) {
+        AppLog.i(AppLog.T.PEOPLE, "resetting people table");
+        dropTables(db);
+        createTables(db);
+    }
+
+    public static void save(Person person) {
+        save(person, getWritableDb());
+    }
+
+    public static void save(Person person, SQLiteDatabase database) {
+        ContentValues values = new ContentValues();
+        values.put("person_id", person.getPersonID());
+        values.put("local_blog_id", person.getLocalTableBlogId());
+        values.put("user_name", person.getUsername());
+        values.put("first_name", person.getFirstName());
+        values.put("last_name", person.getLastName());
+        values.put("display_name", person.getDisplayName());
+        values.put("avatar_url", person.getAvatarUrl());
+        values.put("role", Role.toKey(person.getRole()));
+        database.insertWithOnConflict(PEOPLE_TABLE, null, values, SQLiteDatabase.CONFLICT_REPLACE);
+    }
+
+    public static void savePeople(List<Person> peopleList) {
+        for (Person person : peopleList) {
```

Comment:
You're right, I should have wrapped that in a transaction. Still a bit unfamiliar with some practices. [#526113e](https://github.com/wordpress-mobile/WordPress-Android/pull/3916/commits/526113e35663902489178f49706e01d63aaf9a9a) should fix it, if I got it right. Thanks!